index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap" rel="stylesheet">
+<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.9.0/css/all.css" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header header-home main-grid">
+
+        <div class="header-content-1">
+            <div class="nav-1">
+                <ul class="nav-list-1">
+                    <li class="nav-item-1"><a href="#" class="nav-link-1"><i class="fab fa-twitter"></i></a></li>
+                    <li class="nav-item-1"><a href="#" class="nav-link-1"><i class="fab fa-facebook-square"></i></a></li>
+                    <li class="nav-item-1"><a href="#" class="nav-link-1">English</a></li>
+                    <li class="nav-item-1"><a href="#" class="nav-link-1">MyPage</a></li>
+                    <li class="nav-item-1"><a href="#" class="nav-link-1">Logout</a></li>
+                </ul>
+            </div>
+            </div>
+        </div>
+        <div class="header-content">
+            <a class="logo-link" href="index.html">
+                <img src="images/logo.png" alt="" class="logo">
+            </a>
+            <button aria-label="Open navigation" class="open-nav">&#9776;</button>
+            <nav class="nav">
+                <button aria-label="Close navigation" class="close-nav">&times;</button>
+                <ul class="nav-list">
+                    <li class="nav-item"><a href="index.html" class="nav-link current">home</a></li>
+                    <li class="nav-item"><a href="about.html" class="nav-link">about us</a></li>
+                    <li class="nav-item"><a href="contact.html" class="nav-link">contact</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+<main>
+    <section class="hero main-grid">
+        <h1 class="hero-title">The only marketing agency on <span class="accent-color">a mission from God</span>, fully dedicated to increase your web traffic. </h1>
+        <p class="hero-subtitle">An SEO Company, Creating Custom <strong>SEO</strong> Services, <strong>PPC </strong> Management And <strong>Internet Marketing Services</strong> Since 2019.</p>
+        <a href="index.html" class="btn btn-primary">contact</a>
+    </section>
```

Comment:
- [ ]  Kindly ensure your hero section matches the design guideline regarding the background, typography, color, etc👍🏽. In a more professional working environment, It is imperative to stick to design as much as possible to avoid any issues as regards clients satisfaction 😄 

Design Guildeline 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/190708841-a7e0bbc0-25b3-457f-82f9-46fb968836c1.png)


Your Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/190708909-edca2a96-c4ca-4fb1-94d7-412f3167de67.png)